site/_posts/2017-09-25-last-week-of-summer.md

```diff
@@ -0,0 +1,87 @@
+
+
+## Featured
+
+Some statistics for last week:
+  
+31 PRs merged to the [manageiq][manageiq PRs merged] repository,
+
+27 PRs merged to the [manageiq-ui-classic][manageiq-ui-classic PRs merged] repository,
+          
+25 PRs merged to the [manageiq-ui-service][manageiq-ui-service PRs merged] repository,
+
+plus more across the providers' repositories.
+
+
+## Improved
+
+The Greenkeeper bot continues to create PRs to maintain up to date dependencies for the Service UI - see this week's list [here](https://github.com/ManageIQ/manageiq-ui-service/pulls?utf8=✓&q=is%3Apr+is%3Amerged+label%3Agreenkeeper+base%3Amaster+merged%3A"2017-09-18+..+2017-09-24"+sort%3Acreated-desc).
+
+[PR 979](https://github.com/ManageIQ/manageiq-ui-service/pull/979) refactors the CollectionsApi for extensibility (infinite options) and conciseness.<br>
+
+The dialog generation for orchestration template was modified in [PR 15982](https://github.com/ManageIQ/manageiq/pull/15982) to rely on the templates to provide the deployment options.<br> 
+
+[PR 16011](https://github.com/ManageIQ/manageiq/pull/16011) adds 'description' to the list of automate value fields, allowing it to be updated by an automate method.<br> 
+
+## Fixed
+
+To allow the Azure SSA job to succeed, a number of timeouts and workers' memory were increased in [PR 16016](https://github.com/ManageIQ/manageiq/pull/16016).
+
+After [PR 115995](https://github.com/ManageIQ/manageiq/pull/15995), the request to destroy linked events is queued to prevent the UI hanging when deleting an instance of MiqServer.
+
+## New
+
+[This PR](https://github.com/ManageIQ/manageiq/pull/15911) exposes the Custom Button visibility and enablement based on the evaluation of MiqExpressions set for these on each button.
+
+Custom Buttons have been added to the Generic objects in [PR 15980](https://github.com/ManageIQ/manageiq/pull/15980).
+
+YAML format content is now accepted as an OrchestrationTemplate [PR 298](https://github.com/ManageIQ/manageiq-providers-amazon/pull/298).
+
+After the Automate [PR 15951,](https://github.com/ManageIQ/manageiq/pull/15951) the manual placement selection will support only the use of "VMware clusters" and only the use of "VMware folders".
+
+We are getting ready for Ruby 2.4 - [PR15987](https://github.com/ManageIQ/manageiq/pull/15987) replaces the deprecated Fixnum|Bignum with integer.
+
+[Here](https://github.com/ManageIQ/manageiq/pull/14475) is the initial work for putting the documentation that exists in the source code into a browsable format.
+
+
+## Deleted
+
+The deletion of the week, [PR16005](https://github.com/ManageIQ/manageiq/pull/16005) - removes 'format_by_(class|column)', as these are no longer used.
+
+
+## Product Tools
+
+Two very useful product tools were merged this week: 
+
+[One](https://github.com/ManageIQ/manageiq/pull/15982) to replicate a server settings to other servers.
```

Comment:
Should this be [PR 15990](https://github.com/ManageIQ/manageiq/pull/15990)?

Reply:
@cybette  - fixed

Reply:
@cybette - yes, thanks for noticing, fixed